libraries/ui-library/src/utils/execution-control.ts

```diff
@@ -33,8 +31,10 @@ export const debounce = (callback: Function, timeout = DEFAULT_DEBOUNCE_FAST) =>
  * @param timeout
  */
 export const debounceEvent = (event: EventEmitter, timeout: number): EventEmitter => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const original = (event as any)._original || event;
-  const emit = debounce(original.emit.bind(original), timeout) as (any) => CustomEvent;
+  // @typescript-eslint/no-explicit-any
```

Comment:
Hm, the following line doesn't have an `any`. Couldn't this comment be removed?

Reply:
Good catch 👍